### DISCUSSION TITLE
26 hour pytorch tutorial deep neural nets

### DISCUSSION THREAD

**Discussion by shivaanshgarg** [mrdbourke/pytorch-deep-learning#463] [Q&A]
02. PyTorch Neural Network Classification

Hey Fellas!
I have covered the portion upto chapter 1. I wanted to learn pytorch , not for classification tasks, but for fitting experimental data into a neural network and get predictions. Is that covered in the course? If so, where?

**Comment by Igbo-princee** [comment]
At the beginning, "01 PyTorch workflow".
